include/fastdds/rtps/attributes/ServerAttributes.h

```diff
@@ -131,7 +177,7 @@ const char* const DEFAULT_ROS2_MASTER_URI = "ROS_DISCOVERY_SERVER";
  * @return true if parsing succeeds, false otherwise (or if the list is empty)
  */
 RTPS_DllAPI bool load_environment_server_info(
-        std::string list,
+        const std::string& list,
```

Comment:
This breaks ABI. I'm fine with it, but wanted to note it.

Reply:
👍